dll/shellext/mydocs/CMyDocsDropHandler.cpp

```diff
@@ -157,7 +157,12 @@ CMyDocsDropHandler::Drop(IDataObject *pDataObject, DWORD dwKeyState,
     fileop.pFrom = pszzSrcList;
     fileop.pTo = szzDir;
     fileop.fFlags = FOF_ALLOWUNDO | FOF_FILESONLY | FOF_MULTIDESTFILES | FOF_NOCONFIRMMKDIR;
-    SHFileOperationW(&fileop);
+    int res = SHFileOperationW(&fileop);
+    if (res)
```

Comment:
Replace `return hr;` with this block and `return S_OK;` too.

Reply:
Wth are you talking about @SergeGautherie ?

Reply:
Not understood. Marked as resolved unless clarified.